lib/service/storage/elasticsearch.js

```diff
@@ -49,6 +50,9 @@ const CHILD_MAPPING_PROPERTIES = ['type'];
 const HIDDEN_COLLECTION = '_kuzzle_keep';
 const PRIVATE_PREFIX = '%';
 const PUBLIC_PREFIX = '&';
+const INDEX_PREFIX_POSITON_IN_INDEX = 0;
+const INDEX_PREFIX_POSITON_IN_ALIAS = 1;
+const ALIAS_PREFIX = '@'; // @todo next major release: Add ALIAS_PREFIX in FORBIDDEN_CHARS
```

Comment:
This deserves an official announcement, doesn't it?

Reply:
No, I thought about it and that's why I've chosen to have the `index_prefix` coming between the `alias_prefix` and `index.collection`. Since the `index_prefix` is already forbidden and always there, we should not confused an `@` contained in and index or collection and the proper `alias_prefix` thanks to its position.
But still, I will check if I missed something and add it to the concerns array above